getusermedia.html

```diff
@@ -4817,8 +4817,9 @@ <h2>Methods</h2>
                   <p>If the constraint is <dfn>required</dfn>
                   (<var>constraintValue</var>
                   either contains one or more members named 'min', 'max', or
-                  'exact', or is itself a bare value and bare values are to be
-                  treated as 'exact'), and the <a>settings dictionary</a>'s
+                  'exact', or is itself a bare value in an advanced
+                  ConstraintSet and bare values in advanced CS are to be treated
+                  as 'exact'), and the <a>settings dictionary</a>'s
```

Comment:
bare values in advanced CS are always treated as exact
```suggestion
                  'exact', or is itself a bare value in an advanced
                  ConstraintSet), and the <a>settings dictionary</a>'s
```